### PR TITLE
Docker fixes

### PR DESCRIPTION
**Pull request by anthonybilinski**
- [x] Commits follow our [git commit guidelines](https://github.com/qTox/qTox/blob/master/CONTRIBUTING.md#git-commit-guidelines)

---
This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/6458)

### CODE REVIEW

File: windows/cross-compile/build.sh [hunk 1]
```diff
@@ -105,12 +109,13 @@ mkdir -p "${QTOX_PREFIX_DIR}"
 cp qtox.exe $QTOX_PREFIX_DIR
 cp -r /export/* $QTOX_PREFIX_DIR
 
-# Run tests (only on Travis)
+# Run tests
 set +u
-if [[ -n "$TRAVIS_CI_STAGE" ]]
+if [[ $RUN_TESTS -ne 0 ]]
 then
   export WINEPATH='/export;/windows/bin'
```
Comment (nurupo) [inline windows/cross-compile/build.sh:116]: Does it really work with such `WINEPATH`?
Based on my findings in https://github.com/qTox/qTox/pull/6413#discussion_r781066906 it shouldn't, but it seems like you were able to run tests successfully?

**Comment by nurupo** [comment]
While we are at it, could we copy the source code for the debug build? https://github.com/qTox/qTox/pull/6413#discussion_r778661734

File: windows/cross-compile/build.sh (hunk 1 above)
Comment (anthonybilinski) [reply windows/cross-compile/build.sh:116]: All I can say is that the tests do run yes, I have no problem with them running locally or in CI.

**Comment by anthonybilinski** [comment]
I'm not really convinced of the benefit either. I understand the use case of gdb and the helper scripts, and those seem quite useful for new users or just users who don't want to deal with installing tools on Windows but regarding your point of:
> Including qTox the source code is useful for gdb to generate nice printouts of where the crash happens in the code and for the more advanced users (or even developers themselves) debugging why the crash is happening.

I feel like the nice printout doesn't add anything for the issue report, since with debug symbols that it already has we'll already have a fully visible back trace, visible variables, and the correct file and line number. It would be nice for a more advanced user who wants to investigate, but at that point I figure they'd want to browse the code rather than just see the gdb printouts, by either cloning it or even just browsing on github, and they could still follow along with gdb's output. It also just doesn't seem very standard for even debug applications to ship with bundled source code.

I don't think there's a big downside with doing it, package size I don't think is a big concern given it's the debug version, but I'm still leaning towards leaving it out.

**Review by nurupo — APPROVED** [review]

**Review by anthonybilinski — COMMENTED** [review]
Approving review: :lgtm_strong: 

> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/6458#-:-Mw80oYk5a6oNuOZ9_Hx:b-c1zd3t)* status: :shipit: complete! 1 of 1 LGTMs obtained

**Pull request merged**